home-open-api/src/main/resources/yaml/openapi.yaml

```diff
@@ -1107,6 +1111,410 @@ paths:
           $ref: '#/components/responses/NotFound'
         default:
           $ref: '#/components/responses/InternalServerError'
+
+  /houses/{house_id}/apartments:
+    post:
+      tags:
+        - apartment
+      summary: "Create apartment"
+      description: Create new Apartment in the house
```

Comment:
> In description we use 'The endpoint does something...' so here it will be - The endpoint creates a new Apartment in the house

Proposed changes have been made